src/aks-preview/azext_aks_preview/_params.py

```diff
@@ -301,6 +301,7 @@ def load_arguments(self, _):
         c.argument('message_of_the_day')
         c.argument('gpu_instance_profile', arg_type=get_enum_type(gpu_instance_profiles))
         c.argument('workload_runtime', arg_type=get_enum_type(workload_runtimes), default=CONST_WORKLOAD_RUNTIME_OCI_CONTAINER)
+        c.argument('enable_namespace_resources', help='Enables namespace as an ARM resource')
```

Comment:
we can't use ARM, we need to expand to Azure Resource Manager. So let's modify full string to "Enables sync of namespaces as Azure Resource Manager resources"

Reply:
Done